src/compas/artists/robotmodelartist.py

```diff
@@ -301,8 +331,8 @@ def scale_link(self, link, transformation):
         """
         self._scale_link_helper(link, transformation)
 
-        if self.attached_tool_model:
-            self._scale_link_helper(self.attached_tool_model.root, transformation)
+        for tool in self.attached_tool_models.values():
+            self._scale_link_helper(tool, transformation)
```

Comment:
not sure, but isn't this supposed to be `tool.root` to match what was happening before the change?